pages/login.js

```diff
@@ -0,0 +1,91 @@
+import Head from 'next/head';
+import loginStyles from '@app/styles/login.module.css';
+import {
+  Header,
+  Segment,
+  Form,
+  Button,
+  Checkbox,
+  Breadcrumb,
+} from 'semantic-ui-react';
+import cx from 'classnames';
+import LabeledInput from '@app/components/pieces/labeledInput';
+
+export default function login() {
+  return (
+    <>
+      <Head>
+        <title>Sign In - ConnectOrlando</title>
+        <meta
+          name="description"
+          content="Sign in to your ConnectOrlando account"
+        />
+      </Head>
+      <main className={loginStyles.centerMain}>
+        <Segment basic className={loginStyles.segment}>
+          <Header className={loginStyles.loginHeader} as="h1">
+            Log In
+          </Header>
+
+          <Form>
+            <LabeledInput label="Email" placeholder="name@example.com" />
+            <LabeledInput label="Password" placeholder="****************" />
+          </Form>
+
+          <Form.Field className={loginStyles.verticalSpacing}>
+            <Button
+              type="submit"
+              fluid
+              color="blue"
+              className={cx(loginStyles.loginButton, loginStyles.capitalize)}
+            >
+              log in
+            </Button>
+          </Form.Field>
+          <Form.Field
+            className={cx(
+              loginStyles.verticalSpacing,
+              loginStyles.toOppositeSides
+            )}
+          >
+            <Checkbox
+              className={loginStyles.rememberMeButton}
+              label="Remember me"
+            ></Checkbox>
+            <Breadcrumb.Section
+              className={loginStyles.forgotPasswordBreadcrumb}
+              a
+              href="https://cdn.wallpapersafari.com/74/46/dr3t5f.jpg"
+              link
+            >
+              Forgot Password?
+            </Breadcrumb.Section>
```

Comment:
I like the BreadCrumb component more. Is there any particular reason why it would be better to use the anchor tag () instead besides the "look"? Also is the anchor tag "look" preferred?